haystack/nodes/retriever/_embedding_encoder.py

```diff
@@ -125,6 +157,22 @@ def embed_documents(self, docs: List[Document]) -> List[np.ndarray]:
         passages = [[d.meta["name"] if d.meta and "name" in d.meta else "", d.content] for d in docs]  # type: ignore
         return self.embed(passages)
 
+    def train(self, training_data: List[Dict[str, Any]], batch_size: int = 16) -> None:
```

Comment:
For training I'd wish to be able to pass in some more hyperparameters like learning rate, warmup steps, ...

Reply:
Yeah, we can add that.